docs/reference/contributing/target/bootloader.md

```diff
@@ -0,0 +1,18 @@
+## Bootloader
+
+To enable firmware update capability via bootloader on a new target instructions as well as an example of how to do it can be found in the mbed OS 5 handbook.
```

Comment:
Hmm.. this statement isn't very clear to me.. bootloader is only required to enable FW update on the device via Pelion DM.. it's not done "via" the BL - sounds more like there are other options too (which is not the case today)..so, something like "please find instructions and an example in the handbook to update the FW using the bootloader" sounds more like setting the scene.. @AnotherButler : can you please amend this to make it more an introduction / set the scene type.. thanks..